libraries/AP_Logger/AP_Logger_config.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <AP_HAL/AP_HAL_Boards.h>
```

Comment:
We gained this include? looks identical apart from that.

Reply:
This *must* be included if you want your `#ifndef`s to work properly :-)   This is where the stuff from hwdef.h comes in, for example.

Reply:
Does that mean they didn't work before?